components/sponsors/SponsorCard.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div v-show="!!logoUrl" :class="classObject">
+        <img :src="logoUrl" class="object-contain w-full" />
```

Comment:
```suggestion
        <img :src="logoUrl" alt="" class="object-contain w-full" />
```

Provide the required attribute for `<img>`